examples/megacmd/megacmdexecuter.cpp

```diff
@@ -3098,7 +3140,12 @@ void MegaCmdExecuter::executecommand(vector<string> words, map<string, int> *clf
         mtxSyncMap.lock();
         if (words.size() == 3)
         {
-            string path = fsAccessCMD->expansePath(&words[1]);
+            string path;
+            string localrelativepath;
+            string localabsolutepath;
+            fsAccessCMD->path2local(&words[1], &localrelativepath);
+            fsAccessCMD->expanselocalpath(&localrelativepath, &localabsolutepath);
+            fsAccessCMD->local2path(&localabsolutepath, &path);
```

Comment:
This could also go into a private function for better readability. e.g:
string path;
expansePath(&words[1],&path);